glide.lock

```diff
@@ -1015,17 +1017,6 @@ imports:
   version: 5bd2802263f21d8788851d5305584c82a5c75d7e
 - name: github.com/quobyte/api
   version: cb10db90715b14d4784465d2fa3b915dfacc0628
```

Comment:
I expected to see a hash/date change in this file's header... did you manually fix up the output of `update-deps.sh`?